sklearn/tests/test_common.py

```diff
@@ -377,3 +378,15 @@ def test_transformer_n_iter():
 
         if hasattr(estimator, "max_iter") and name not in external_solver:
             yield check_transformer_n_iter, name, estimator
+
+def test_get_params_invariance():
```

Comment:
Doing ``include_other=True`` in ``test_all_estimators`` would be nice, and allowing the other estimators in this test, would be nice, too, but we can leave that for the future.